lib-new/src/lib_new/__init__.py

```
@@ -0,0 +1,3 @@
+from cowsay import cow
+def hello(message: str) -> str:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)